<?xml version="1.0"?>
<launch>

        <include file="$(find astra_camera)/launch/astra.launch"  />

        <node name="object_detect" pkg="sophon_robot" type="FaceRecog_node.py" output="screen">
                <param name="camera_name" value="camera" />
        </node>

</launch>
